### PR TITLE
Add round amount after DCC to gateway account setting

### PR DESCRIPTION
**Pull request by dara123**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccount.yaml [hunk 1]
```diff
@@ -256,6 +256,11 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  roundAmountAfterForcedDcc:
+    type: boolean
+    default: false
+    description: |-
+      Round DCC quote amount to the nearest non-decimal amount.
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccount.yaml:263]: What is the round mode: half up or half down or round to even or round odd?

File: openapi/components/schemas/GatewayAccount.yaml [hunk 2]
```diff
@@ -256,6 +256,11 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  roundAmountAfterForcedDcc:
```
[the remark this answers is not in the document]
Comment (dara123) [reply openapi/components/schemas/GatewayAccount.yaml:259]: maybe `forcedDccRoundAmount` ?

[the remark this answers is not in the document]
Comment (dara123) [reply openapi/components/schemas/GatewayAccount.yaml:259]: we have `dccForceCurrency` maybe `dccForceRounding`

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccount.yaml [hunk 3]
```diff
@@ -256,6 +256,10 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  dccForceRounding:
+    type: boolean
+    default: false
+    description: Specifies that DCC quote amounts must be round half up to the nearest non-decimal amount.
```
Comment (colinbird) [inline openapi/components/schemas/GatewayAccount.yaml:262]:
```suggestion
    description: Specifies that DCC quote amounts must be rounded half up to the nearest non-decimal amount.
```

**Review by colinbird — APPROVED** [review]

**Review by noon-dawg — DISMISSED** [review]
Looks good on the docs side

**Review by ashkarpetin — DISMISSED** [review]

**Review by colinbird — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccount.yaml [hunk 4]
```diff
@@ -256,6 +256,10 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  dccForceRounding:
+    type: boolean
+    default: false
+    description: Specifies that DCC quote amounts must be rounded half up to the nearest non-decimal amount.
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccount.yaml:262]:
```suggestion
    description: Specifies that DCC quote amounts must be rounded half up to the nearest whole number.
```
?

`nearest non-decimal amount` seems incorrect

Comment (noon-dawg) [reply openapi/components/schemas/GatewayAccount.yaml:262]: Good point

**Review by noon-dawg — APPROVED** [review]

**Comment by dara123** [comment]
@Rebilly/php please review

File: openapi/components/schemas/GatewayAccount.yaml [hunk 5]
```diff
@@ -256,6 +256,10 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  dccForceRounding:
+    type: boolean
+    default: false
```
Comment (nanobash) [inline openapi/components/schemas/GatewayAccount.yaml:261]: Perhaps example should be also included?

```suggestion
    default: false
    example: false
```

**Review by lexerom — APPROVED** [review]

**Pull request merged**